lib/index.js

```diff
@@ -123,12 +124,37 @@ Table.prototype.toString = function (){
       //sum of columns, borders
       return sum + ((colWidths.length*2) - 1);
     }();
-
-    if(terminal_width < required_width){
-      var new_col_width = (terminal_width - (colWidths.length * 2) -1) / colWidths.length;
-      new_col_width = Math.floor(new_col_width);
-      colWidths = colWidths.map(function(){
-        return new_col_width;
+    
+    if(terminal_width <= required_width){
+      var remainingTerminalWidth = terminal_width - (colWidths.length * 2);
```

Comment:
Maybe it is good to follow the author convention for naming locals with underscores.